test/specs/register.spec.ts

```diff
@@ -77,12 +68,9 @@ describe("Page elements test", () => {
     await LoginPage.continueButton.click();
     await LoginPage.masterPasswordInput.setValue(generatedPassword);
     await LoginPage.loginWithMasterPasswordButton.click();
-  });
-  it("should check that login to your account was successful", async () => {
-    let LoginPage = new Login();
     const currentUrl = await browser.getUrl();
     expect(currentUrl).to.include(`vault`);
-    await LoginPage.navigationBar.waitForDisplayed({ timeout: 5000 });
+    await LoginPage.sidebarNavigation.waitForDisplayed({ timeout: 5000 });
```

Comment:
Pauses must be gone, or if they are must-have, add a comment why

Reply:
done